CDTDatastore/HTTP/CDTSessionCookieInterceptor.m

```diff
@@ -77,21 +67,6 @@ - (CDTHTTPInterceptorContext *)interceptRequestInContext:(CDTHTTPInterceptorCont
 }
 
 /**
- We assume a 401 means that the cookie we applied at request time was rejected. Therefore
- clear it and tell the HTTP mechanism to retry the request. For all other responses, there's
- nothing for this interceptor to do.
- */
-- (CDTHTTPInterceptorContext *)interceptResponseInContext:(CDTHTTPInterceptorContext *)context
```

Comment:
Shouldn't this still override to check the `403` `credentials_expired` case? Although it looks like this may never have been handled in this library.